packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -69,12 +70,11 @@ export const AddToCartModal = () => {
             <ModalOverlay />
             <ModalContent
```

Comment:
On desktop, the modal scrolling is not the same as what's defined in the mocks: https://www.figma.com/proto/bp84cmaeZpGrtzIfwtq7yW/%5B244%5D-PWA-Kit-Product-Sets?page-id=579%3A24470&node-id=644%3A24551&viewport=-2434%2C1779%2C0.26&scaling=scale-down-width

Reply:
And on desktop, the modal's max-height is supposed to be 90% of the viewport height, as mentioned in the mocks (on this [page](https://www.figma.com/proto/bp84cmaeZpGrtzIfwtq7yW/%5B244%5D-PWA-Kit-Product-Sets?page-id=579%3A24470&node-id=579%3A27730&viewport=-2434%2C1779%2C0.26&scaling=scale-down-width)).

Reply:
@vmarta According to https://chakra-ui.com/docs/components/modal#modal-overflow-behavior , `scrollBehavior` should control for the modal overflowing beyond the viewport, however, when I set this to `inside` the behavior doesn't work as expected. Have you had experience working with this before or know why this would be occurring?

Reply:
@yunakim714 I don't have experience with that.. outside of my playing with it right now. 

It looks to me that for the `inside` value to work, we'll need to constraint the height of the modal. So that that it's within the viewport, and the thing that overflows (and becomes scrollable) is the modal's content.